test/test_manifest.json

```diff
@@ -5226,6 +5226,12 @@
        "type": "eq",
        "annotations": true
     },
+    {  "id": "issue13487",
+       "file": "pdfs/issue13487.pdf",
+       "md5": "80044ae1ce3fb8c748d6a72f868d5de1",
+       "rounds": 1,
```

Comment:
When you've addressed the previous comment, please ensure that you're only testing *one* page rather than the entire document; make use of the `firstPage`/`lastPage` properties as necessary (see examples of that in the `test_manifest.json` file).